WebService.Test/v1/Controllers/SimulationsControllerTest.cs

```diff
@@ -175,6 +177,27 @@ public void ItReturnsTheSimulationById()
             Assert.Equal(simulation.Id, result.Id);
         }
 
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
+        public void ItInvokesGetSimulationStatisticsOnceOnGetAsync()
+        {
+            // Arrange
+            const string ID = "1";
+            var simulation = this.GetSimulationById(ID);
+
+            this.simulationsService
+                .Setup(x => x.GetAsync(ID))
+                .ReturnsAsync(simulation);
+
+            // Act
+            var result = this.target.GetAsync(ID).Result;
```

Comment:
use `var result = this.target.GetAsync(ID).CompleteOrTimeout().Result` to make sure the test runs with a limited timeout and doesn't run in the background

Reply:
fixed